www/js/types/diaryTypes.ts

```diff
@@ -3,19 +3,18 @@
  As much as possible, these types parallel the types used in the server code. */
 
 import { BaseModeKey, MotionTypeKey } from '../diary/diaryHelper';
+import { ServerData, LocalDt } from './serverData';
+import { FeatureCollection, Feature, Geometry } from 'geojson';
```

Comment:
Looks like this is working well - but make sure that `@types/leaflet` (or whatever you did `npm i` for to be able to access `geojson`) is added to `package.serve.json` and `package.cordovabuild.json`.

In this repo we don't commit `package.json` directly, so we have this extra step whenever adding packages.